Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9.0-alpine3.12
+
+COPY ./requirements.txt ./requirements.txt
+COPY ./config.json ./config.json
+COPY ./export.json ./export.json
+COPY ./credentials.json ./credentials.json
+COPY ./transformer.py ./transformer.py
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+RUN pip install -r requirements.txt
+
+ENTRYPOINT ["python", "transformer.py"]
```

Comment:
It would be cool to be able to pass additionnal arguments.
I've found it is possible using the `$@` to do so.
Can you add that ?

Reply:
It's not necessary, all the arguments after the image name in the 'docker run' command are automatically arguments of the entry point.

Reply:
Hello,

It tried yesterday the `$@` with entry point, but it doesn't work and it's not necessary. As @zigarn said, docker forward all arguments after to the entry point :) 

So, to be sure, the expected command used by the final user should be something like that:

```
docker run -v $CONFIG_ON_HOST:$CONFIG_IN_CONTAINER conference-hall-to-calendar $ARGUMENTS
```

So depending on your OS, the executed command change:

On mac:
```
docker build . -t conference-hall-to-calendar
docker run -v $PWD/config-default.json:/config.json  conference-hall-to-calendar
```

On windows:
```
docker build . -t conference-hall-to-calendar
docker run -v %cd%/config-default.json:/config.json  conference-hall-to-calendar
```

I'll add this on the readme :)

Reply:
The readme is up to date

Reply:
From the direction of this image use, may be better to add the path options (shouldn't they be arguments as they are mandatory, btw?) in the entrypoint and then inject them through volumes:

```dockerfile
ENTRYPOINT ["python", "transformer.py", "--input", "/export.json", "--configuration", "/config.json"]
```

```shell
docker run -v $PWD/my-config.json:/config.json -v $PWD/my-export.json:/export.json conference-hall-to-calendar
```